### PR TITLE
Document installation via the repository

### PR DESCRIPTION
**Pull request by BenPope**
```sh
helm repo add vectorizedio https://vectorizedio.github.io/helm-charts/
helm install --namespace redpanda --create-namespace redpanda vectorizedio/redpanda
```

Signed-off-by: Ben Pope <ben@vectorized.io>

### CODE REVIEW

**Review by RafalKorepta — APPROVED** [review]
Thanks for that!

File: README.md [hunk 1]
```diff
@@ -13,6 +13,13 @@ This is the Helm Chart for [Redpanda](https://vectorized.io).
 
 ## Installation
 
+### Using the repository
+
+```sh
+helm repo add vectorizedio https://vectorizedio.github.io/helm-charts/
```
Comment (rkruze) [inline README.md:19]: I believe the docs use `redpanda` as the repo name, should we use `redpanda` here or change the docs to be the same?

Comment (BenPope) [reply README.md:19]: But then it becomes: `helm install redpanda/redpanda` 

vectorizedio matches the url and is closer to `docker run vectorized/redpanda`

And the repository is "helm-charts", so other Vectorized wares could be listed there.

But I hold no strong opinion.

Comment (BenPope) [reply README.md:19]: Are there docs for this already?

Comment (BenPope) [reply README.md:19]: Oh, I see `helm repo add redpanda https://charts.vectorized.io/`

Comment (rkruze) [reply README.md:19]: There are, we can update the docs. https://vectorized.io/docs/quick-start-kubernetes

Comment (BenPope) [reply README.md:19]: I hold no strong opinions on this! Was just trying to do some tidy up.

Comment (BenPope) [reply README.md:19]: If the chart is already published, we can probably close this, or link the K8s quick start guide instead. One single mechanism is better.

**Pull request closed without merging**